docs/guides/upgrading.rst

```diff
@@ -224,14 +224,27 @@ Messages will no longer get the metadata 'msg' for newly created messages. This
 Removed Classes
 ---------------
 
+ - ``ElggInspector``
  - ``FilePluginFile``: replace with ``ElggFile`` (or load with ``get_entity()``)
 
 Removed keys available via ``elgg_get_config()``
 ------------------------------------------------
 
+ - ``allowed_ajax_views``
+ - ``dataroot_in_settings``
+ - ``externals``
+ - ``externals_map``
+ - ``i18n_loaded_from_cache``
+ - ``language_paths``
+ - ``pagesetupdone``
+ - ``registered_tag_metadata_names``
+ - ``simplecache_enabled_in_settings``
+ - ``translations``
+ - ``viewpath``
  - ``views``
+ - ``view_path``
```

Comment:
Seems like a reasonable set to remove. I don't recall any being on @iionly's list...